packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -0,0 +1,685 @@
+              ' to ' +
+              chalk.hex('#e00000')(`25`) +
+              ' words.',
+          ]),
+        receivedErrorMessage(error),
+      );
+    }
+    failedTests.push('has-docblock');
+  },
+
+  'exports-component': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+    const { componentPath, displayName } = testInfo;
+    const fileName = path.basename(componentPath);
+
+    // Message Description: There are no matching exported components that match the displayName.
+    //
+    // It appears that "Component" doesn't have a export in: "componentPath"
+    // Here are the available exported component's from your file: "componentFile"
+    // To Resolve this issue:
+    // 1. Make sure that your component's "fileName" file contains an export for "displayName".
+    // 2. Check to see if you component uses export default and consider enabling useDefaultExport in your
+    // isConformant test.
+    console.log(
+      defaultErrorMessage(
+        `exports-component`,
+        displayName,
+        'export in: ' + paragraph() + chalk.green.italic(componentPath),
+      ) +
+        resolveErrorMessages([
+          `Make sure that your component's ` +
+            chalk.hex('#e00000')(fileName) +
+            ' file contains an export for ' +
+            chalk.hex('#e00000')(displayName) +
+            '.',
+          'Check to see if you component uses export default and consider enabling' +
+            chalk.hex('#e00000')(' useDefaultExport ') +
+            'in your isConformant test.',
+        ]),
+      receivedErrorMessage(error),
+    );
+    failedTests.push('exports-component');
+  },
+
+  'component-renders': (componentInfo: ComponentDoc, testInfo: IsConformantOptions, error: string) => {
+    const { displayName, componentPath, requiredProps } = testInfo;
+    const typesFile = componentPath.replace('tsx', 'types.ts');
+
+    // Message Description: Handles scenario where a invalid return is received and the user provides requiredProps.
+    //
+    // It appears that displayName doesn't have a valid return.
+    // It currently is receiving the requiredProps: "requiredProps"
+    // Check to see if you are including all of the required props in your types file: "typesFile"
+    // To Resolve this issue:
+    // 1. Make sure that your are including all of the required props to render in isConformant.
+    // 2. Make sure that your component's "displayName".base.tsx file contains a valid return statement.
+    // 3. Check to see if your component works as expected with mount and safeMount.
+    if (requiredProps) {
+      const formatRequiredProps = () => {
+        const results = [];
+        for (const libName of Object.keys(requiredProps)) {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          results.push(libName + `: ${(requiredProps as any)[libName] + ', '}`);
+        }
+        return results.join('\n');
```

Comment:
Some suggestions here:
- If you're creating an array and immediately filling it with a 1-1 mapping from another array, it's better to just use `map`
- In Node code you can use `Object.entries()` if you need both the key and value (it's ES6 so not allowed in the browser yet)
- Would be better to just use a template string to build the whole line
- It's probably better to add the commas when joining so you don't get an extra comma after the last prop
```suggestion
        return Object.entries(requiredProps)
          .map(([propName, propValue]) => `${propName}: ${propValue}`)
          .join(',\n');
```